src/common/Collision/Models/WorldModel.cpp

```diff
@@ -621,7 +626,7 @@ namespace VMAP
         {
             //if (fread(&chunkSize, sizeof(uint32), 1, rf) != 1) result = false;
 
-            if (result && fread(&count, sizeof(uint32), 1, rf) != 1) { result = false; }
+            if (fread(&count, sizeof(uint32), 1, rf) != 1) { result = false; }
             if (result) { groupModels.resize(count); }
```

Comment:
The condition for this if statement is also redundant as we're already inside the `if (result && readChunk(rf, chunk, "GMOD", 4))` if statement

Reply:
@steenburgh no, because if it gets inside the first `if`` then `false` will be assigned to `result`

Reply:
👍Right, I missed that

Reply:
@steenburgh thanks for checking my PR!